lib/src/sdk.ts

```diff
@@ -335,22 +327,26 @@ export class ExchangeSDK {
     // 2 - Ask for payload creation
     this.logger.log("Call getSellDestinationAccount");
     const { recipientAddress, amount, binaryPayload, signature } =
-      await getSellPayload(
-        deviceTransactionId,
-        account.address,
-        BigInt(fromAmountAtomic.toString()),
-      );
+      await getSellPayload(deviceTransactionId, account.address, BigInt(0));
```

Comment:
The `amount` selected by the user need to be passed again to the `getSellPayload` to give a chance to the LiveApp to double check the value.

Reply:
I agree with the current use done by the Coinify LiveApp.
However, we need to be more flexible about the use of the `ExchangeSDK`. Some other context might send us a first sell amount, which will be adjust later.

So my idea is to let the call with
```js
await getSellPayload(
        deviceTransactionId,
        account.address,
        BigInt(fromAmountAtomic.toString()),
      );
```
And also let the previous check on the "enough amount"

But adding the on we have done below too. As you fix it, we can have a different amount, thus that need to be check

Reply:
yeah that makes sense, i've made the changes